content/questions/foundations/question-108.md

```diff
@@ -0,0 +1,16 @@
+---
+archetype: "questions"
+title: "Question 108"
+question: "What is one of the main benefits for using a Personal Access Token (PAT) instead of a standard username and password in GitHub authentication?"
+draft: false
+---
+
+
+> https://learn.microsoft.com/en-us/training/modules/github-introduction-administration/3-how-github-authentication-works
+1. [ ] PATs can be managed by the organization and enterprise.
+> The study guide does not mention that organizations and enterprises can manage PATs.  Those entities can manage other forms of authentication, such as LDAP, SAML, and SSH keys.
```

Comment:
This is a practice exam, just explain why the answer is wrong here. Don't mention what is or isn't covered in study guides/courses/articles

```suggestion
> PAT's as the name suggests, are personal. Organizations and Enterprises can manage other forms of authentication, such as LDAP, SAML, and SSH keys.
```